### ISSUE TITLE
No automatic syncing of ResourceCache

### ISSUE THREAD

**Issue by bmah888** [esnet/netshell#16]
Got this stack trace:
```
java.lang.Exception: Stack trace
	at java.lang.Thread.dumpStack(Thread.java:1365)
	at net.es.netshell.api.ResourceCache$Writer.run(ResourceCache.java:47)
	at java.lang.Thread.run(Thread.java:745)
java.lang.Exception: Stack trace
	at java.lang.Thread.dumpStack(Thread.java:1365)
	at net.es.netshell.api.ResourceCache$Limiter.run(ResourceCache.java:93)
	at java.lang.Thread.run(Thread.java:745)
```
Something bad happened to the threads that are supposed to sync the resource cache.  As a result, changes don't get written to the database.  The workaround for now is to manually invoke `Resource.getCache().sync()`.

**Comment by lomaxfrog** [comment]
Problem was due to a recent change allows JUnit test to run even when Netshell is not running. Problem is fixed in commit ade84b3d7a0e95175374b43d857c80587a8275f8

Bruce, could you please verify that this is issue is fixed and if it is mark it as resolve ?

Thanks

**Comment by bmah888** [comment]
Looks to me, thanks!  Closing.
